test/fixture/test.js

```diff
@@ -102,3 +104,47 @@ test('logLevel', async t => {
 	t.is(rendererLogs.length, 1);
 	t.regex(rendererLogs[0], /error/);
 });
+
+test('hookConsole', async t => {
+	t.context.app = new Application({
+		path: electron,
+		args: ['index_hookConsole.js']
```

Comment:
I moved some of the main process test logic into separate files, because other tests' logging was polluting `app.client.getMainProcessLogs()`.

I thought this might be a better approach than letting the tests affect each other. What do you think

Reply:
If we do decide to go with this approach, then it’s probably better to change the `beforeEach` and `afterEach` hooks